site-new/blog/2019-05-28-first-blog-post.md

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
So you prefer adding these meaningless posts and replacing them one by one. Is that correct?

Reply:
They were auto-generated and will be replaced :)